iModelCore/ECDb/Tests/NonPublished/ECDbMetaSchemaECSqlTestFixture.cpp

```diff
@@ -98,7 +98,7 @@ void ECDbMetaSchemaECSqlTestFixture::AssertSchemaDef(ECSchemaCR expectedSchema,
             ASSERT_EQ(m_ecdb.Schemas().GetClass("ECDbMeta", "ECSchemaDef")->GetId(), val.GetId<ECClassId>()) << "ECSchemaDef.ECClassId";
             ASSERT_STREQ("ClassId", colInfo.GetProperty()->GetAsPrimitiveProperty()->GetExtendedTypeName().c_str()) << "ECSchemaDef.ECClassId";
             }
-        else if (colName.EqualsI("Name"))
+        else if (colName.EqualsI("Name") || colName.EqualsI("Name_1"))

```

Comment:
@simasjar can you elaborate on the scenario where the colName turns into Name_1 here?
How does it happen since no views are involved?

Reply:
It doesn't turn into Name_1 with the current implementation, removed those checks